File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/question/AnswerDocumentTest.java
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.question;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;

/**
 * Unit test class for {@link AnswerDocument}
 */
public class AnswerDocumentTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(AnswerDocument.class)
				.verify();
	}

}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/question/AnswerDocumentTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.question

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.Test

/**
 * Unit test class for [AnswerDocument]
 */
class AnswerDocumentTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(AnswerDocument::class.java)
				.verify()
	}

}